src/index.ts

```diff
@@ -65,13 +63,11 @@ class ServerlessSSMPublish {
       'ssmPublish:upsertParams': this.hookWrapper.bind(this, this.updateParams.bind(this)),            // tslint:disable-line:no-unsafe-any
 
       // Serverless lifecycle events
-      'after:package:createDeploymentArtifacts': async () => { // check if this is the best place to call our plugin
+      'after:deploy:deploy': async () => { // check if this is the best place to call our plugin
```

Comment:
could you give a short explanation why the change in events?

Reply:
Sure.
Basically I think it should always have been run after deploy, but I was testing things by running `sls package`, which cuts out before getting to the deploy lifecycle event.

The main reason I'm leaning towards after deploy is because that way you won't need to rollback SSM variables that might have been created before a failed deployment (and that's the description you gave the repo on github).

Reply:
sounds like a wise choice 👍